fetch/api/request/destination/fetch-destination.https.html

```diff
@@ -232,6 +270,24 @@
   });
 }, 'HTMLLinkElement with rel=preload and as=style fetches with a "style" Request.destination');
 
+// HTMLLinkElement with rel=preload and as=json - json destination
+promise_test(async t => {
+  await new Promise((resolve, reject) => {
+      let node = frame.contentWindow.document.createElement("link");
+      node.rel = "preload";
+      node.as = "json";
+      if (node.as != "json") {
+        resolve();
+      }
+      node.onload = resolve;
+      node.onerror = reject;
+      node.href = "dummy.json?t=2&dest=json";
+      frame.contentWindow.document.body.appendChild(node);
+  }).catch(() => {
```

Comment:
in theory, there should be no need to catch here. Just returning the new promise at line 275 should do the trick.